### PR TITLE
[OSCI] Remove unused tutorials

### LINKED ISSUE #4448: Remove unused `tutorials`
The repository has a lot of content under
* src/plugins/home/server/tutorials
* src/plugins/home/public/application/components/tutorial
* src/plugins/home/public/assets/tutorials
... among others

This are inaccessible and serve no purpose. We should get rid of them.

### PR DESCRIPTION
**Pull request by CMDWillYang**
### Description

Removed unused tutorials from:
src/plugins/home/server/tutorials
src/plugins/home/public/assets/tutorials/logos

### Issues Resolved

fixes #4448
### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/home/server/services/tutorials/tutorials_registry.ts [hunk 1]
```diff
@@ -36,7 +36,6 @@ import {
   ScopedTutorialContextFactory,
 } from './lib/tutorials_registry_types';
 import { tutorialSchema } from './lib/tutorial_schema';
-import { builtInTutorials } from '../../tutorials/register';
```
Comment (CMDWillYang) [inline src/plugins/home/server/services/tutorials/tutorials_registry.ts:39]: this is an unused import, and also the only usage of tutorials under src/plugins/home/server/tutorials

**Comment by CMDWillYang** [comment]
In the last commit, I removed unused svg assets under src/plugins/home/public/assets/tutorials/logos. 

To verify the those were not used, I mainly searched the whole repo for usage of these logos, and I've found that those in the logos folder were not used(see some screenshots below). 

Screenshot 2023-10-17 at 9 47 09 PM
Screenshot 2023-10-17 at 9 43 49 PM

There does remain four png files under the tutorials folder (not tutorials/logos) that appear to be still used.
Screenshot 2023-10-17 at 9 54 16 PM

I know in the issue it's suggested to remove tutorials under src/plugins/home/public/application/components/tutorial. However, after some digging it appears most are still heavily used in other parts of the project, so I left those in for now.

If there's any suggestion on what more to remove, tips on how and where to search for additional used tutorials, or just feedback in general it would be greatly appreciated.

And of course, thanks so much to Josh for helping me here and at office hours to not completely drown in the sea of OpenSearch codebase :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -117,6 +117,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Console] Migrate `/lib/!autocomplete/` module to TypeScript ([#4150](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4150))
 - [Dashboard] Restructure the `Dashboard` plugin folder to be more cohesive with the project ([#4575](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4575))
 - Refactor logo usage to centralize and optimize assets and improve tests ([#4702](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4702))
+- Remove unused tutorials ([#5212] (https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5212))
```
Comment (joshuarrrr) [inline CHANGELOG.md:120]:
```suggestion
- [Home] Remove unused tutorials ([#5212](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5212))
```

**Review by joshuarrrr — APPROVED** [review]
LGTM! Minor suggestion on the changelog entry.

**Comment by Willie-The-Lord** [comment]
LGTM as well!

**Review by joshuarrrr — APPROVED** [review]

File: src/plugins/home/server/services/tutorials/tutorials_registry.ts (hunk 1 above)
Comment (joshuarrrr) [reply src/plugins/home/server/services/tutorials/tutorials_registry.ts:39]: @CMDWillYang Either in this PR, or as a standalone followup, can you also delete this comment: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/0ff9bc57b477b1349564299095f4a00c90e0359d/src/plugins/home/server/services/tutorials/tutorials_registry.ts#L97-L100

**Comment by joshuarrrr** [comment]
> I know in the issue it's suggested to remove tutorials under src/plugins/home/public/application/components/tutorial. However, after some digging it appears most are still heavily used in other parts of the project, so I left those in for now.
>
> If there's any suggestion on what more to remove, tips on how and where to search for additional used tutorials, or just feedback in general it would be greatly appreciated.

@CMDWillYang Can you open a new issue for discussing what to do with these? I agree that we should probably handle them separately, and can add some additional context.

**Review by kavilla — APPROVED** [review]

**Pull request merged**